src/tests/Loader/classloader/MethodImpl/generics_override1.ilproj

```diff
@@ -1,6 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk.IL">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
+
+    <!-- Testing TypeLoad/MissingMethod exceptions in situations that are expensive to detect -->
```

Comment:
We'd have to add complete validation that all virtual methods make sense on a type in a spot where type system exceptions can be caught. We've considered these not to be worth the compiler throughput hit.